### ISSUE TITLE
Automatically scroll the container

### ISSUE THREAD

**Issue by rraziel** [d3lm/ngx-drag-to-select#69]
Hello,

This module works great, however a really nice feature would be the ability for the drag selection to scroll the container when going out of (visible) bound.

Unless such a feature already exists and I'm using the component incorrectly, which is entirely possible.

Thanks

**Comment by d3lm** [comment]
Hey @rraziel, thanks! I am glad and happy that it works nice for you.

This is indeed a nice feature to have. Would you be up to take a look at this and send a PR? I am more than happy to assist you and help you with anything that comes up. I would really appreciate any help.

Any remarks on your end @timdeschryver?

**Comment by timdeschryver** [comment]
Yea, why didn't we think of this 😅 
I think it would be a great addition!

**Comment by d3lm** [comment]
@rraziel So how do you think should this behave when there are multiple scroll containers? Cause you could have the page that is scrollable as well as the select-container which might be scrollable too. You think that the container is enough because I don't want to make this too complex and try to keep the lib as lightweight as possible.

Ideas @timdeschryver?

**Comment by timdeschryver** [comment]
Personally, I would expect that the direct parent would scroll with the mouse movement. 
If every container scrolls, it would look a bit funny imho 😅

**Comment by d3lm** [comment]
Yep totally.

**Comment by rraziel** [comment]
I'm not sure how reliable this would be - especially considering people may also use extra components for infinite scroll - but I expect finding the first scrollable ancestor might work?

**Comment by d3lm** [comment]
@rraziel Yep, that might be enough.

**Comment by suhailsalim** [comment]
@d3lm Any update on this ?
Thanks. :)

**Comment by d3lm** [comment]
Not yet. Haven't had the time to work on this. As this library is open source I would appreciate any help and maybe @suhailsalim you wanna take a stab at this and send a PR if this is a pressing issue?

**Comment by d3lm** [comment]
Or of course anyone else is also welcome to send a PR to introduce this feature.

**Comment by kinnu69** [comment]
any update on this feature?

**Comment by d3lm** [comment]
@kinnu69 No, I don't have the capacity right now to work on this and would welcome a PR.
